src/views/dialog/warning.php

```diff
@@ -0,0 +1,32 @@
+<?php
+/**
+ * Warn Dialog View Template
+ * The confirmation template for tribe-dialog.
+ *
+ * Override this template in your own theme by creating a file at [your-theme]/tribe/dialogs/warn.php
+ *
+ * @since TBD
+ *
+ * @package Tribe
+ * @version TBD
+ */
+
+/** @var \Tribe\Dialog\View $dialog_view */
+$dialog_view = tribe( 'dialog.view' );
+// grab allthevars!
+$vars        = get_defined_vars();
+?>
+<?php $dialog_view->template( 'button', $vars, true ); ?>
+<script data-js="<?php echo esc_attr( 'dialog-content-' . $id ); ?>" type="text/template" >
+	<div <?php tribe_classes( $content_classes  ) ?>>
+		<?php if ( ! empty( $title ) ) : ?>
+			<h2 <?php tribe_classes( $title_classes ) ?>><?php echo esc_html( $title ); ?></h2>
+		<?php endif; ?>
+
+		<?php echo $content; ?>
```

Comment:
Do we want to escape this?

Reply:
That content will often have HTML

Reply:
Or wp_kses

Reply:
^ Oops haha, this last comment was left there before I saw yours. Yes, I realized that this could have HTML when I was checking the other PR. Maybe wp_kses could make sense here then?